karma.conf.js

```diff
@@ -0,0 +1,74 @@
+// Karma configuration
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha', 'chai'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+        'bower_components/angular/angular.js',
+        'bower_components/angular-route/angular-route.js',
+        'bower_components/angular-sanitize/angular-sanitize.js',
+        'bower_components/google-code-prettify/src/prettify.js',
+        'node_modules/angular-mocks/angular-mocks.js',
+        './app/**/*.js',
+        './tests/**/*Spec.js'
```

Comment:
Yes, I add it later. It matches files like:
 - tests/specs/directives/BsNavActiveDirectiveSpec.js
 - tests/specs/directives/PrettyprintDirectiveSpec.js
 - tests/specs/directives/TableRowLinksDirectiveSpec.js

I think this is in order as I had decided that `tests` folder will hold the testing files and they'd be ending with `Spec.js` like how our python ones end with `Test.py` when making this commit.